packages/frontend-shared/cypress/e2e/support/e2eSupport.ts

```diff
@@ -243,6 +253,39 @@ function loginUser (userShape: Partial<AuthenticatedUserShape> = {}) {
   })
 }
 
+function findBrowsers (options: SetFoundBrowsersOptions = {}) {
```

Comment:
Wrote a helper to make browser stubbing a little easier. By default it will stub out all supported browsers + electron, but the options allow you to either filter that list or provide your own set. 

How we do the stubbing may change with the completion of #19347.